### ISSUE TITLE
Consider splitting out a package for submitters

### ISSUE THREAD

**Issue by spbnick** [kernelci/kcidb#376]
The current kcidb package contains everything that's running in the cloud, administrative tools, and the code using for submitting data. Consider splitting out a separate package that could be used for submitting data only, and perhaps for querying. This won't get rid of our most-cumbersome dependency - the forked jq.py package, but might still be beneficial.

**Comment by shivam-Purohit** [comment]
@spbnick can I try this out?

**Comment by shivam-Purohit** [comment]
I thought there will be a pattern in functions that are used for submitting data and for querying.

**Comment by spbnick** [comment]
Yeah, it could be a "little" more entangled. Please go ahead and take a look at the other issues.

**Comment by spbnick** [comment]
We could very much use this to speed up deployment of our Cloud Functions, as each of them wouldn't need to build the jq dependency.
